test/bats/onchain-send.bats

```diff
@@ -134,21 +135,33 @@ teardown() {
   [[ "${settled_status}" = "SUCCESS" ]] || exit 1
 }
 
-@test "onchain-send: settle intraledger" {
+@test "onchain-send: settle intraledger, with no contacts check" {
```

Comment:
with contacts check?

Reply:
So interestingly I don't think we have contacts for `send-onchain` use-case. I'm not sure if that was deliberate or not so I captured current behaviour here. The assertion check that contacts _isn't_ added after the send.